package.json

```diff
@@ -5,8 +5,6 @@
   "private": true,
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "dev": "webpack --mode development",
```

Comment:
- [ ] It looks like you delete the development mode, I kindly invite you to add it and test if everything is fine. `"dev": "webpack --mode development",` 🐶